platform/action/core/platform-action-core-impl/src/main/java/ddf/action/impl/AbstractActionRegistry.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.action.impl;
+
+import ddf.action.Action;
+import ddf.action.ActionProvider;
+import ddf.action.ActionRegistry;
+import ddf.action.MultiActionProvider;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class AbstractActionRegistry implements ActionRegistry {
```

Comment:
Neither child of this class does anything beyond calling the parent constructor. Why not call this class something like `ActionRegistryImpl` and just use it directly?